### DISCUSSION TITLE
Stereographic projection along [011] pole

### DISCUSSION THREAD

**Discussion by pvan-sr** [mtex-toolbox/mtex#1725] [Ask Anything] [answered]
Hi all,

I am a new MTeX user, apologies if this is a trivial question.
I am trying to plot the stereographic projections of certain plane normals (0,1,1), (0,-1,1),(1,0,1),(-1,0,1),(1,-1,0) and (1,1,0) with respect to the (0,1,1) pole on the XY plane. Is there a way of plotting this directly in MTeX?

Thanks,
Pavan

**Comment by kilir** [comment, marked as answer]
Hi,
I'm not entirely sure if I understand what you'd like to do. Could you try to phrase your question differently?
Cheers,
Rüdiger

**Reply by pvan-sr** [reply]
Hi @kilir,

As I understand currently, the stereographic projections are created using the poles of 001 plane in my cubic system. I was wondering if MTeX could plot the projections with the poles of the 011 plane?
I am trying to recreate this figure on MTeX from this [paper](https://doi.org/10.2355/isijinternational.ISIJINT-2019-448)
![image](https://user-images.githubusercontent.com/123510041/214646391-da36e89f-136a-4192-9792-a0fded6562f0.png)

Thanks,
Pavan

**Reply by kilir** [reply]
Hi,
have a look at `help orientation/plotPDF`, you can simply do `plotPDF(o,h)` where h is a Miller of your choice.
Hope that helps.
Rüdiger
